modules/kmm-validation-kickoff.adoc

```diff
@@ -6,21 +6,10 @@
 [id="kmm-validation-kickoff_{context}"]
 = Validation kickoff
 
-Preflight validation is triggered by creating a `PreflightValidationOCP` resource in the cluster. This spec contains two fields:
+Preflight validation is triggered by creating a `PreflightValidationOCP` resource in the cluster.
+This spec contains two fields:
 
-[source,terminal]
-----
-type PreflightValidationOCPSpec struct {
-	// releaseImage describes the OCP release image that all Modules need to be checked against.
-	// +kubebuilder:validation:Required
-	ReleaseImage string `json:"releaseImage"` <1>
-	// Boolean flag that determines whether images build during preflight must also
-	// be pushed to a defined repository
-	// +optional
-	PushBuiltImage bool `json:"pushBuiltImage"` <2>
-}
-----
+`kernelVersion`:: The version of the kernel that the cluster is upgraded to. This field is required.
```

Comment:
This section also requires revert. In Downstream we specify the `releaseImage` , not the `kernelVersion`

Reply:
@qbarrand Done. Please check and /lgtm if all changes are complete.